### PR TITLE
bugfix for unwanted char in modelica object names

### PR DESCRIPTION
**Pull request by triccyx**
Bugfix for unwanted char in urdf objects when translated to Modelica objects

### CODE REVIEW

File: src/sdf-modelica-lib/src/sdf_modelica.cpp [hunk 1]
```diff
@@ -118,8 +119,16 @@ bool modelicaFromSDF(sdf::SDFPtr sdf,
   data["fileName"] = options.originalFilename;
 
   // modelName
-  data["modelName"] = modelElement->Get<std::string>("name");
-
+  const std::string tmpModelName=modelElement->Get<std::string>("name");
+  data["modelName"] = tmpModelName;
+  
+  //Modelica Language specification Version 3.3 Apendix B
```
Comment (traversaro) [inline src/sdf-modelica-lib/src/sdf_modelica.cpp]: Minor typos: 
Apendix --> Appendix 
Space between // and Modelica

**Comment by triccyx** [comment]
Done

File: README.md [hunk 2]
```diff
@@ -256,3 +257,7 @@ This model is render in [OpenModelica](https://openmodelica.org/) as:
 ![twoLinks model](https://user-images.githubusercontent.com/1857049/40165638-c92b527c-59bc-11e8-94a3-6253e3fe5417.png)
 
 The flange indicated with the `joint_1_2` label is a classical Modelica acausal flange of type [`Modelica.Mechanics.Rotational.Interfaces.Flange_a`](https://doc.modelica.org/help/Modelica_Mechanics_Rotational_Interfaces.html#Modelica.Mechanics.Rotational.Interfaces.Flange_a), so the model can be interfaced with arbitrary Modelica models.  
+
+## Naming
+Due to the Modelica naming conventions, the urdf file that is to be translated, should contain model name, link name and joint name with only alphanumeric or underscore characters.
```
Comment (traversaro) [inline README.md]: the URDF or SDF file that is translated should contain model name, link names and joint names with

File: README.md [hunk 3]
```diff
@@ -256,3 +257,7 @@ This model is render in [OpenModelica](https://openmodelica.org/) as:
...
+See also Modelica language specification Appendix B https://www.modelica.org/documents
```
Comment (traversaro) [inline README.md]: Please add a point after the sentence.

File: README.md [hunk 4]
```diff
@@ -259,5 +259,5 @@ This model is render in [OpenModelica](https://openmodelica.org/) as:
 The flange indicated with the `joint_1_2` label is a classical Modelica acausal flange of type [`Modelica.Mechanics.Rotational.Interfaces.Flange_a`](https://doc.modelica.org/help/Modelica_Mechanics_Rotational_Interfaces.html#Modelica.Mechanics.Rotational.Interfaces.Flange_a), so the model can be interfaced with arbitrary Modelica models.  
 
 ## Naming
-Due to the Modelica naming conventions, the urdf file that is to be translated, should contain model name, link name and joint name with only alphanumeric or underscore characters.
-See also Modelica language specification Appendix B https://www.modelica.org/documents
\ No newline at end of file
+Due to the Modelica naming conventions, the urdf file that is translated, should contain model name, link name and joint name with only alphanumeric or underscore characters.
```
Comment (traversaro) [inline README.md]: Please mention also SDF files and use plurals for link and joint names, as mentioned in https://github.com/robotology-playground/sdf-modelica/pull/12#discussion_r195347740 , thanks!

**Comment by traversaro** [comment]
Merged, thanks!

**Pull request merged**